### PR TITLE
Simplify construct_runtime syntax

### LINKED ISSUE #634: Simplify `construct_runtime` syntax
### Context
We can use the new `construct_runtime` syntax where the default types can be left out, it's recommended as it's less error-prone.

Like instead of 
```
System: frame_system::{Pallet, Call, Storage, Config, Event<T>} = 0,
```
we can write
```
System: frame_system = 0,
```

### Task
Simplify and adjust the syntax

---
:heavy_check_mark: Please set appropriate **labels** and **assignees** if applicable.

### PR DESCRIPTION
**Pull request by ziming-zung**
resolves #634

### CODE REVIEW

File: runtime/litentry/src/lib.rs [hunk 1]
```diff
@@ -759,32 +759,30 @@ construct_runtime! {
...
+		Multisig: pallet_multisig = 4,
+		Proxy: pallet_proxy = 5,
+		Preimage: pallet_preimage = 6,
 
 		// Token related
-		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 10,
-		Vesting: pallet_vesting::{Pallet, Call, Storage, Event<T>, Config<T>} = 11,
+		Balances: pallet_balances = 10,
+		Vesting: pallet_vesting = 11,
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 12,
```
Comment (Kailai-Wang) [inline runtime/litentry/src/lib.rs]: This one is left out I think, or intentional?

File: runtime/litmus/src/lib.rs [hunk 2]
```diff
@@ -793,32 +793,30 @@ construct_runtime! {
...
+		Multisig: pallet_multisig = 4,
+		Proxy: pallet_proxy = 5,
+		Preimage: pallet_preimage = 6,
 
 		// Token related
-		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 10,
-		Vesting: pallet_vesting::{Pallet, Call, Storage, Event<T>, Config<T>} = 11,
+		Balances: pallet_balances= 10,
+		Vesting: pallet_vesting= 11,
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 12,
```
Comment (Kailai-Wang) [inline runtime/litmus/src/lib.rs]: same as above

File: runtime/litmus/src/lib.rs [hunk 3]
```diff
@@ -832,34 +830,34 @@ construct_runtime! {
...
-		PolkadotXcm: pallet_xcm::{Pallet, Call, Storage, Event<T>, Origin, Config} = 51,
-		CumulusXcm: cumulus_pallet_xcm::{Pallet, Call, Event<T>, Origin} = 52,
-		DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 53,
-		XTokens: orml_xtokens::{Pallet, Call, Storage, Event<T>} = 54,
+		XcmpQueue: cumulus_pallet_xcmp_queue = 50,
+		PolkadotXcm: pallet_xcm = 51,
+		CumulusXcm: cumulus_pallet_xcm = 52,
+		DmpQueue: cumulus_pallet_dmp_queue = 53,
+		XTokens: orml_xtokens = 54,
 		Tokens: orml_tokens::{Pallet, Call, Storage, Event<T>} = 55,
```
Comment (Kailai-Wang) [inline runtime/litmus/src/lib.rs]: This one is missing

File: runtime/moonbase/src/lib.rs [hunk 4]
```diff
@@ -767,32 +767,30 @@ construct_runtime! {
...
+		Multisig: pallet_multisig = 4,
+		Proxy: pallet_proxy = 5,
+		Preimage: pallet_preimage = 6,
 
 		// Token related
-		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 10,
-		Vesting: pallet_vesting::{Pallet, Call, Storage, Event<T>, Config<T>} = 11,
+		Balances: pallet_balances = 10,
+		Vesting: pallet_vesting = 11,
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 12,
```
Comment (Kailai-Wang) [inline runtime/moonbase/src/lib.rs]: same as above

File: runtime/moonbase/src/lib.rs [hunk 5]
```diff
@@ -806,29 +804,29 @@ construct_runtime! {
...
-		PolkadotXcm: pallet_xcm::{Pallet, Call, Storage, Event<T>, Origin, Config} = 51,
-		CumulusXcm: cumulus_pallet_xcm::{Pallet, Call, Event<T>, Origin} = 52,
-		DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 53,
-		XTokens: orml_xtokens::{Pallet, Call, Storage, Event<T>} = 54,
+		XcmpQueue: cumulus_pallet_xcmp_queue = 50,
+		PolkadotXcm: pallet_xcm = 51,
+		CumulusXcm: cumulus_pallet_xcm = 52,
+		DmpQueue: cumulus_pallet_dmp_queue = 53,
+		XTokens: orml_xtokens = 54,
 		Tokens: orml_tokens::{Pallet, Call, Storage, Event<T>} = 55,
```
Comment (Kailai-Wang) [inline runtime/moonbase/src/lib.rs]: same as above

File: runtime/rococo/src/lib.rs [hunk 6]
```diff
@@ -797,32 +797,30 @@ construct_runtime! {
...
+		Multisig: pallet_multisig = 4,
+		Proxy: pallet_proxy = 5,
+		Preimage: pallet_preimage = 6,
 
 		// Token related
-		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 10,
-		Vesting: pallet_vesting::{Pallet, Call, Storage, Event<T>, Config<T>} = 11,
+		Balances: pallet_balances = 10,
+		Vesting: pallet_vesting = 11,
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 12,
```
Comment (Kailai-Wang) [inline runtime/rococo/src/lib.rs]: same as above

File: runtime/rococo/src/lib.rs [hunk 7]
```diff
@@ -836,34 +834,34 @@ construct_runtime! {
...
-		PolkadotXcm: pallet_xcm::{Pallet, Call, Storage, Event<T>, Origin, Config} = 51,
-		CumulusXcm: cumulus_pallet_xcm::{Pallet, Call, Event<T>, Origin} = 52,
-		DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 53,
-		XTokens: orml_xtokens::{Pallet, Call, Storage, Event<T>} = 54,
+		XcmpQueue: cumulus_pallet_xcmp_queue = 50,
+		PolkadotXcm: pallet_xcm = 51,
+		CumulusXcm: cumulus_pallet_xcm = 52,
+		DmpQueue: cumulus_pallet_dmp_queue = 53,
+		XTokens: orml_xtokens = 54,
 		Tokens: orml_tokens::{Pallet, Call, Storage, Event<T>} = 55,
```
Comment (Kailai-Wang) [inline runtime/rococo/src/lib.rs]: same as above

File: runtime/litmus/src/lib.rs [hunk 8]
```diff
@@ -793,32 +793,30 @@ construct_runtime! {
...
+		Scheduler: pallet_scheduler = 2,
+		Utility: pallet_utility = 3,
+		Multisig: pallet_multisig = 4,
+		Proxy: pallet_proxy = 5,
+		Preimage: pallet_preimage = 6,
 
 		// Token related
-		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 10,
-		Vesting: pallet_vesting::{Pallet, Call, Storage, Event<T>, Config<T>} = 11,
+		Balances: pallet_balances= 10,
```
Comment (Kailai-Wang) [inline runtime/litmus/src/lib.rs:806]: space before `=`

File: runtime/litmus/src/lib.rs [hunk 9]
```diff
@@ -793,32 +793,30 @@ construct_runtime! {
...
-		Treasury: pallet_treasury::{Pallet, Call, Storage, Config, Event<T>} = 13,
+		Treasury: pallet_treasury = 13,
 
 		// Governance
-		Democracy: pallet_democracy::{Pallet, Call, Storage, Config<T>, Event<T>} = 21,
-		Council: pallet_collective::<Instance1>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 22,
-		CouncilMembership: pallet_membership::<Instance1>::{Pallet, Call, Storage, Event<T>, Config<T>} = 23,
-		TechnicalCommittee: pallet_collective::<Instance2>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 24,
-		TechnicalCommitteeMembership: pallet_membership::<Instance2>::{Pallet, Call, Storage, Event<T>, Config<T>} = 25,
+		Democracy: pallet_democracy= 21,
```
Comment (Kailai-Wang) [inline runtime/litmus/src/lib.rs]: space before `=`

**Review by Kailai-Wang — COMMENTED** [review]
Generally looks good, but some of my questions inline

**Comment by ziming-zung** [comment]
It is intentional. Because these pallet have  `GenesisConfig` and we don't write in chain_spec configuration . If left out the types, there are some errors during compiling.

**Review by Kailai-Wang — APPROVED** [review]

**Pull request merged**